src/ZoneServer/World/Maps/Map.cs

```diff
@@ -190,8 +191,20 @@ private void Disappearances()
 			lock (_monsters)
 				toDisappear = _monsters.Values.Where(a => a.DisappearTime < now).ToList();
 
-			foreach (var monster in toDisappear)
+			foreach (var monster in toDisappear) {
+
+				// If this is a blue orb summon, inform the summoner that it expired
```

Comment:
This comment seems misleading, because what's happening is not that the character gets informed in some way. All that happens is that the orb summon variables are reset. Missing comments are usually preferable over incorrect ones.

Reply:
I've adjusted the comment to be more clear about what it actually does.